src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+import { StyledModal } from './Modal.style';
+
+export const Modal: React.FC = () => {
+	const [open, setOpen] = useState(false);
+
+	const handleOpen = () => setOpen(true);
+	// const handleClose = () => setOpen(false);
+
+	return (
+		<div data-testid="Modal">
+			<button onClick={handleOpen}>Open modal</button>
```

Comment:
Non-blocking: It would be great if you could use the Redback UI [Button component](https://redback-operations.github.io/redback-ui/?path=/story/components-button--default) here.

Reply:
I have tried my best to import the Redback UI Button component inside the modal component, but I'm currently having trouble figuring out how to pass in the necessary arguments. The best I could manage was to recreate the button component within the modal file, but this resulted in a lot of redundant code. Could we please create a separate ticket for this so that someone else can also work on finding a solution?

Reply:
Yep that sounds like a good solution. :) I completely agree we should avoid redundant code, and if there's any problem with the button component that would be a good task for someone to do separately. (And if not, maybe it'll be a quick win for someone to implement it into your story.)